src/pages/index.hbs

```diff
@@ -5,6 +5,23 @@
 </div>
 
 <div class="drizzle-o-Layout__content drizzle-o-Layout__content--centered drizzle-b-LongFormText">
+<div class="drizzle-c-Alert">
+    <div class="drizzle-c-Alert__icon">
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512">
+        <path d="M569.517 440.013C587.975 472.007 564.806 512 527.94 512H48.054c-36.937 0-59.999-40.055-41.577-71.987L246.423 23.985c18.467-32.009 64.72-31.951 83.154 0l239.94 416.028zM288 354c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/>
+      </svg>
+    </div>
+    <p class="drizzle-c-Alert__message sprk-u-mbn">
+      We are currently working on a new version of our documentation site! The new site is
```

Comment:
I'm not sure how useful it is to send people to the new docs before we have all of the components in the navigation. Instead, what if we say something like this:

We are working on a new version of our documentation site! You can see a preview of the new developer docs now for Angular, React, and HTML (link each of those).

The new docs are under active development and may change at any time. please reach out, yada yada.

Reply:
Ahh great point! I forgot that we haven't added all the component page content yet. I made the update. Thank you! @yodasw16